web/src/app/profile/approved-researcher/components/ApprovedResearcherForm.css

```
@@ -18,9 +18,14 @@
   margin-bottom: 1rem;
 }
 
+.input-checkbox {
+  margin: 0.5rem;
+}
+
 .submit-button {
```

can we [bem](https://getbem.com/naming/) this por favor
```suggestion
.button--submit {
```